server/src/helpers/utils.helper.ts

```diff
@@ -52,3 +53,13 @@ export function sanitizeInput(value: string) {
 export function lowercaseString(value: string) {
   return value?.toLowerCase();
 }
+
+export async function dbManagerWrapper(operation: (...args) => any, manager: EntityManager): Promise<any> {
```

Comment:
Can we reword this from `dbManagerWrapper` to `dbTransactionWrap` ?

Does refactoring this to the one below work?

```
export async function dbManagerWrapper(operation: (...args) => any, manager = getManager()): Promise<any> {
  await manager.transaction(async (manager) => {
    return await operation(manager);
  });
 }
```
The advantage being one wouldn't have to always pass the second param and have it wrapped in a transaction

Reply:
Not working, if `manager` exists we have already started transactions for previous queries  `await manager.transaction` throws error

Reply:
Okay, thought it would result in nested transactions 🤔 
Ignore then 👍🏾